src/components/MissionVision.jsx

```
@@ -0,0 +1,216 @@
+import React, { useState, useRef } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)